### DISCUSSION TITLE
Github Pages Not Deploying My Website

### DISCUSSION THREAD

**Discussion by Mat-foucher** [community/community#51487] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello,

I have been using github pages _without_ using a custom domain (yet) for my personal website. Typically I have found that I have the option to deploy my site to the URL hosted by github itself, but I see no such option in my repository anymore. I have an index.html file but there is no option to publish my site onto github and I have no idea why it is not working. Any help would be greatly appreciated, thanks.

**Comment by mrepol742** [comment]
Just go to the repository `Settings` then `Pages`.
Note: You cannot published pages on private repository if you are not in Github Pro or Github Entreprise.
After it just choose between deploy from a branch or Github Action.
